bin/dev/kube-ready-state-check.sh

```diff
@@ -78,62 +78,63 @@ function having_category() {
 
 echo "Testing $(green "${category}")"
 
-# swap accounting in /proc/cmdline
+# swap accounting in /proc/cmdline.
 if having_category node ; then
     grep -wq "swapaccount=1" /proc/cmdline
     status "swapaccount enable"
 
-    # docker info should not show aufs
+    # `docker info` should not show aufs.
     docker info 2> /dev/null | grep -vwq "Storage Driver: aufs"
     status "docker info should not show aufs"
 fi
 
-# kube-dns shows 4/4 ready
+# kube-dns shows 4/4 ready.
 if having_category kube ; then
     kubectl get pods --namespace=kube-system --selector k8s-app=kube-dns 2> /dev/null | grep -Eq '([0-9])/\1 *Running'
     status "kube-dns should be running (show 4/4 ready)"
 fi
 
-# tiller-deploy shows 4/4 ready
+# tiller-deploy shows 4/4 ready.
 if having_category kube ; then
     kubectl get pods --namespace=kube-system --selector name=tiller 2> /dev/null | grep -Eq '([0-9])/\1 *Running'
     status "tiller should be running (1/1 ready)"
 fi
 
-# ntp or systemd-timesyncd is installed and running
+# ntp or systemd-timesyncd is installed and running.
 if having_category api node ; then
-    pgrep -x ntpd >& /dev/null || pgrep -x chronyd >& /dev/null || systemctl is-active systemd-timesyncd >& /dev/null
+    (ps x | grep ntpd >& /dev/null) || (ps x | grep chronyd >& /dev/null) || systemctl is-active systemd-timesyncd >& /dev/null
```

Comment:
Oh wow, didn't realize `pgrep` just completely ignored the executable name after 15 characters.  But `grep` here means it will match commands with that substring (e.g. `/usr/sbin/ntpdate`).  And none of the things we're testing for are 15 characters long.